src/components/include/utils/threads/thread.h

```diff
@@ -172,7 +173,11 @@ class Thread {
    */
   void stop();
 
-  void join();
+  /**
+   * @brief Starts join procedure.
+   * @param[in] force_stop If true stop() is called first otherwise is skipped.
+   */
+  void join(const bool force_stop = true);
```

Comment:
@AGaliuzov, fixed in [e1ece13](https://github.com/smartdevicelink/sdl_core/pull/707/commits/e1ece13e6b21765b2acc11b5b50c855c81234443) and [ce7130e](https://github.com/smartdevicelink/sdl_core/pull/707/commits/ce7130e0fa97a34d5fb143ff0764557cee82eb68) regarding enums.